ts-defs-src/test-api/action-options.d.ts

```diff
@@ -174,3 +174,42 @@ interface ResizeToFitDeviceOptions {
      */
     portraitOrientation?: boolean;
 }
+
+interface CookieOptions {
+    /**
+     * Specifies the name of the cookie.
+     */
+    name?: string;
+    /**
+     * Specifies the value of the cookie.
+     */
+    value?: string;
+    /**
+     * Specifies the domain which will be matched with the cookie.
+     */
+    domain?: string;
+    /**
+     * Specifies the path which will be matched with the cookie.
```

Comment:
is it the path *to* the cookie?

Reply:
It indicates a URL path that must exist in the requested URL in order to send the Cookie

Reply:
```suggestion
     * The path attribute of the cookie. The path must exist on the server to send the Cookie header.
```